jobs/build-webapp.groovy

```diff
@@ -642,7 +651,10 @@ def deployAndReport() {
       }
       parallel(jobs);
 
-      uploadGraphqlSafelist();
+      parallel([
+         "create-cloud-run-tags": { createCloudRunTags(); },
+         "update-graphql-safelist": { uploadGraphqlSafelist(); }
```

Comment:
This is fine, but I don't think you need to create a closure just to call it!  You can probably do:
```
"create-cloud-run-tags": createCloudRunTags,
"update-graphql-safelist": uploadGraphqlSafelist,
```
But I don't know groovy very well, so ymmv.

Reply:
Clearly I dont know groovy very well either... So Im following how everything else does it.  For example https://github.com/Khan/jenkins-jobs/blob/62709da2d847bbe1d3dbeebd2668c6b32a939a4c/jobs/deploy-webapp.groovy#L680